File: SurveyBuilderApp/SurveyBuilderApp/Classes/cSurveyQuestions.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Text;

namespace SurveyBuilderApp.Classes
{
    [Foundation.Preserve(AllMembers = true)]
    [DataContract]
    public class cSurveyQuestions
    {
        [DataMember]
        public int QuestionId { get; set; }
        [DataMember]
        public string Question { get; set; }
        [DataMember]
        public string QuestionDescription { get; set; }           
        [DataMember]
        public bool Mandatory { get; set; }
        [DataMember]
        public int Control { get; set; }
        [DataMember]
        public int SurveyId { get; set; }
        [DataMember]
        public DateTime QuestionCreatedDateTime { get; set; }
        [DataMember]
        public DateTime QuestionLastModifiedDateTime { get; set; }
        [DataMember]
        public int DisplayId { get; set; }
        [DataMember]
        public bool Enabled { get; set; }
        public string  QuestionAndNumber
        {
            get
            {
                return string.Format("Q{0}. {1}", DisplayId, Question);
            }
        }
    }
}

File: Survey Builder/Classes/cSurveyQuestions.vb
Public Class cSurveyQuestions
    Public Property QuestionId As Integer
    Public Property Question As String
    Public Property QuestionDescription As String
    Public Property Mandatory As Boolean
    Public ReadOnly Property MandatoryCss As String
        Get
            If Mandatory = True Then Return "QuestionMandatory"
            Return ""
        End Get
    End Property
    Public Property Control As cControl
    Public Property QuestionCreatedDateTime As DateTime
    Public Property QuestionLastModifiedDateTime As DateTime
    Public Property Survey As cSurvey
    Public Property DisplayId As Integer
    Public Property Enabled As Boolean

End Class
